src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,28 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import { Todo, User } from '../../types';
+import { UserInfo } from '../UserInfo';
+
+type TodoWithUser = Todo & {
+  user: User | undefined
+};
```

Comment:
```suggestion
type TodoWithUser = Todo & {
  user?: User
};
```
it's better to move typedefs to separate file